core/unit_test/incremental/Test14_MDRangeReduce.hpp

```diff
@@ -79,6 +79,14 @@ struct MyComplex {
   }
 };
 
+KOKKOS_INLINE_FUNCTION
+MyComplex operator+(const MyComplex& src, const MyComplex& t2) {
+  MyComplex temp;
+  temp._re = src._re + t2._re;
+  temp._im = src._im + t2._im;
+  return temp;
+}
```

Comment:
We need to be careful with that change.  Are changing the reduction requirements on the value type?

Reply:
@dalg24 I added that in because Thrust does needs an overload operator+ for its reductions since it does not do them in place, would it be safer if I put ifdefs KOKKOS_ENABLE_THRUST around the section?